fhttp/http_test.go

```diff
@@ -941,6 +941,24 @@ func TestChangeMaxPayloadSize(t *testing.T) {
 	}
 }
 
+func TestAddHTTPS(t *testing.T) {
+	var tests = []struct {
+		input    string
+		expected string
+	}{
+		{"www.google.com", "https://www.google.com"},
+		{"http://www.google.com", "https://www.google.com"},
+		{"https://www.google.com", "https://www.google.com"},
+	}
```

Comment:
in theory we'd add mixed case e.g `hTTPs://foo.com/` but I don't think it'll pass in that case so not yet

Reply:
To pass it, I added 	pURL = strings.ToLower(pURL) in the beginning of the AddHTTPS method.

Reply:
that would break things (url params are case sensitive)

Reply:
While I was doing that I didn't think the param cases ;). I have added a method and test cases for it as well.